src/main/webapp/about.jsp

```diff
@@ -64,6 +64,14 @@
         This page should also be used to describe the features and improvements
         you've added.
       </p>
+
+      <h1>About Team int ellects;</h1>
+      <h3>Our Members: Ivy, Jose, Marielle, and Silvie</h3>
+      <p>
+        Ivy is a junior at the University of Notre Dame.
+        Jose is studying computer science at Purdue University.
+        Marielle is a computer science student at Syracuse University. She sings, dances, and plays volleyball.
+        Silvie is a computer science and anthropology student at Wellesley College. She is involved with the jazz band and frisbee team.
```

Comment:
Sorry to be annoying, but try to keep your lines shorter than 100 characters (including spaces), otherwise the diff view in GitHub gets a little hard to read.